distributed/utils_test.py

```diff
@@ -2148,6 +2148,12 @@ def ucx_loop():
     ucp.reset()
     loop.close()
 
+    # Reset also Distributed's UCX initialization, i.e., revert the effects of
+    # `distributed.comm.ucx.init_once()`.
+    import distributed.comm.ucx
+
+    distributed.comm.ucx.ucp = None
+
```

Comment:
Aside, I think we should move this fixture to the `test_ucx.py` testfile and make it an `autouse=True` fixture. Then when writing a test we don't have to remember to require it.

Reply:
Ah, OK, scratch that then.